packages/components/pie-modal/src/modal.scss

```diff
@@ -56,7 +56,11 @@
 
     & .c-modal-closeBtn {
         position: absolute;
-        right: var(--dt-spacing-d);
-        top: var(--dt-spacing-d);
+        inset-inline-end: var(--dt-spacing-d);
+        inset-block-start: var(--dt-spacing-d);
     }
 }
+
+dialog::backdrop {
+  background: var(--dt-color-overlay);
```

Comment:
Will revert this change as it doesn't quite work. At the moment the only way this works is by setting the rgb directly. The style also works if we move it inside of the main selector so will move it and point it to the rgb color we use if we can't get this to work correctly.